### DISCUSSION TITLE
processing information

### DISCUSSION THREAD

**Discussion by MMOVEt** [community/community#102191] [GitHub Education]
### Hi everyone,

After entering me email, I've submitted me affiliation certificate, but it seems I'm caught in a bit of a bind. I've given it a go eight times, mind you, and the blasted website seems to be playing hard to get. Even attempted it in the incognito window, and all I'm met with is a rather unhelpful "processing information."

**Comment by muizzhaque** [comment]
Clear your browser cache and cookies: Sometimes, old or corrupted data can cause issues with how a website functions.

Try a different browser: If you’re using Chrome, for example, try using Firefox or Safari instead. This can help determine if the issue is specific to one browser.

Disable browser extensions: Some extensions can interfere with how a webpage loads or functions. Try disabling them to see if that resolves the issue.

Check your internet connection: A slow or unstable internet connection can cause issues with webpages loading or processing information.

Wait and try again later: If the website is experiencing high traffic, it might be slow

**Reply by MMOVEt** [reply]
web, tells Your application cannot be reviewed until you fix the following:
You have an outstanding discount request for @MMOVEt. We will get back to you soon.

Ah. And thank you very much for your advice.

**Reply by muizzhaque** [reply]
You are welcome!

**Reply by muizzhaque** [reply]
Is it has solved, can you mark it as answered!
